libs/core/src/lib/message-box/message-box-semantic-icon/message-box-semantic-icon.component.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectionStrategy, Component, Input, Optional } from '@angular/core';
+import {
+    MESSAGE_BOX_CONFIGURABLE_ELEMENT,
+    MessageBoxConfig,
+    MessageBoxConfigurableElement
+} from '../utils/message-box-config.class';
+
+/**
+ * Message box element representing the semantic icon in the message box header.
+ *
+ * ```html
+ * <fd-message-box-semantic-icon></fd-message-box-semantic-icon>
+ * ```
+ */
+@Component({
+    selector: 'fd-message-box-semantic-icon',
+    template: `<i [ngClass]="'sap-icon--' + this._semanticIcon" role="presentation"></i>`,
```

Comment:
no `this.` required

Reply:
1. It's not up to us to decide if a button should be show or not. There's no button in the designs and in the UX guidelines so if we want a component to be following Fiori3 we should not display such button. 
2. will check this and let you know :)